test/unit/react-imgix.test.jsx

```diff
@@ -648,43 +648,39 @@ describe("When using the component", () => {
           const srcSets = srcSet.split(",").map(v => v.trim());
           const srcSetUrls = srcSets.map(srcSet => srcSet.split(" ")[0]);
           removeFallbackSrcSet(srcSetUrls).forEach(srcSetUrl => {
-            const w = parseParam(srcSetUrl, "w");
-            const h = parseParam(srcSetUrl, "h");
+            const ar = parseParam(srcSetUrl, "ar");
 
-            expect(h).toEqual(Math.ceil(w / arDecimal));
-            expect(w).toBeTruthy();
-            expect(h).toBeTruthy();
+            expect(ar).toBeTruthy();
           });
         });
       };
       [
-        ["1:1", "1"],
-        ["1.1:1", "1.1"],
-        ["1.12:1", "1.12"],
-        ["1.123:1", "1.123"],
-        ["1:1.1", "0.9090909090909091"],
-        ["1:1.12", "0.8928571428571428"],
-        ["1.1:1.1", "1"],
-        ["1.123:1.123", "1"],
-        ["11.123:11.123", "1"]
+        ["1:1"],
+        ["1.1:1"],
+        ["1.12:1"],
+        ["1.123:1"],
+        ["1:1.1"],
+        ["1:1.12"],
+        ["1.1:1.1"],
+        ["1.123:1.123"],
+        ["11.123:11.123"]
       ].forEach(([validAR, validArDecimal]) =>
         testValidAR({
           ar: validAR,
-          arDecimal: validArDecimal
         })
       );
     });
 
     describe("invalid AR", () => {
       const testInvalidAR = ar => {
-        it(`height should not be set when an invalid aspectRatio (${ar}) is passed`, () => {
+        it(`an invalid ar prop (${ar}) will still generate an ar query parameter`, () => {
```

Comment:
@jayeb Typically, passing an invalid `ar` value/format would result in a warning and then ignoring the rest of the aspect ratio operation. 
While I definitely like the extra layers set in place to protect against user error, I don't think we should treat any single imgix parameter differently than the others. For that reason, I think the implementation should still pass through the (incorrect) `ar` value, as is the case with other imgix libraries. 
However, I am still debating as to whether we should keep the warning or not. Thoughts?

Reply:
👍 for the warning (but I'm biased, I wrote it 😂)